codalab/apps/web/models.py

```diff
@@ -2722,3 +2722,21 @@ def filename(self):
 def competitiondump_post_delete_handler(sender, **kwargs):
     comp_dump = kwargs['instance']
     delete_key_from_storage(comp_dump, 'data_file')
+
+class NewsPost(models.Model):
```

Comment:
Would the newsletter app be a better fit than web if the newsletter app is not niche